pkgs/development/interpreters/luajit/default.nix

```diff
@@ -98,6 +103,7 @@ stdenv.mkDerivation rec {
     "DEFAULT_CC=cc"
     "CROSS=${stdenv.cc.targetPrefix}"
     "HOST_CC=${buildStdenv.cc}/bin/cc"
+    "LUA_ROOT=$out"
```

Comment:
1. this is unnecessary, PREFIX is already set
2. makeFlags get quoted at the shell level, so this would have to be `$(out)`
3. i'm not sure it would do anything even if it used the correct syntax

Reply:
I tested and it is not necessary, I removed it.

Reply:
i know it's not necessary because i didn't have it in my competing pull request and that worked fine